drivers/serial/uart_stm32.c

```diff
@@ -962,7 +962,7 @@ void uart_stm32_dma_rx_cb(const struct device *dma_dev, void *user_data,
 		 * called in ISR context. So force the RX timeout
 		 * to minimum value and let the RX timeout to do the job.
 		 */
-		k_delayed_work_submit(&data->dma_rx.timeout_work, K_TICKS(1));
+		k_work_schedule(&data->dma_rx.timeout_work, K_TICKS(1));
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/pull/34128#discussion_r612200943 says the intent was to use reschedule in all cases; this is still schedule?

Reply:
Yes, sorry, I did,  but the push failed. Fixing a credential issue.

Reply:
done